src/app/app-platform/ContentApp.h

```diff
@@ -65,8 +66,9 @@ class DLL_EXPORT ContentApp
     virtual MediaPlaybackDelegate * GetMediaPlaybackDelegate()             = 0;
     virtual TargetNavigatorDelegate * GetTargetNavigatorDelegate()         = 0;
 
-    EmberAfStatus HandleReadAttribute(ClusterId clusterId, AttributeId attributeId, uint8_t * buffer, uint16_t maxReadLength);
-    EmberAfStatus HandleWriteAttribute(ClusterId clusterId, AttributeId attributeId, uint8_t * buffer);
+    chip::Protocols::InteractionModel::Status HandleReadAttribute(ClusterId clusterId, AttributeId attributeId, uint8_t * buffer,
```

Comment:
This is inside `namespace chip`, so the `chip::` is not needed anywhere here, right?

Reply:
right, but considering the scale of the work, it was way easier to simply replace all occurences of EmberAfStatus with a full namespace path instead of looking individually at each header files to see what was the actual scope.